share/spack/gitlab/cloud_pipelines/stacks/radiuss/spack.yaml

```diff
@@ -0,0 +1,128 @@
+# This file is currently being customized for quartz/toss_3_x86_64_ib.
+
+spack:
+  concretization: separately
+  view: false
+  config:
+    concretizer: clingo
+    install_tree:
+      root: /home/software/radiuss
+      padded_length: 512
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  packages::
+    all:
+      permissions:
+        read: world
+      providers:
+        mpi: [mvapich2]   # white238
+        #mpi: [mvapich2, openmpi]  # Is this why craylib-sci shows up?
+        # Spack defaults (since limiting the scope)
+        D: [ldc]
+        awk: [gawk]
+        blas: [openblas, amdblis]
+        daal: [intel-daal]
+        elf: [elfutils]
+        fftw-api: [fftw, amdfftw]
+        flame: [libflame, amdlibflame]
+        gl: [mesa+opengl, mesa18, opengl]
+        glx: [mesa+glx, mesa18+glx, opengl]
+        glu: [mesa-glu, openglu]
+        golang: [gcc]
+        iconv: [libiconv]
+        ipp: [intel-ipp]
+        java: [openjdk, jdk, ibm-java]
+        jpeg: [libjpeg-turbo, libjpeg]
+        lapack: [openblas, amdlibflame]
+        mariadb-client: [mariadb-c-client, mariadb]
+        mkl: [intel-mkl]
+        mpe: [mpe2]
+        #mpi: [openmpi, mpich]
+        mysql-client: [mysql, mariadb-c-client]
+        opencl: [pocl]
+        osmesa: [mesa+osmesa, mesa18+osmesa]
+        pil: [py-pillow]
+        pkgconfig: [pkgconf, pkg-config]
+        rpc: [libtirpc]
+        scalapack: [netlib-scalapack, amdscalapack]
+        sycl: [hipsycl]
+        szip: [libszip, libaec]
+        tbb: [intel-tbb]
+        unwind: [libunwind]
+        yacc: [bison, byacc]
+        uuid: [util-linux-uuid, libuuid]
+  definitions:
+    #- compilers: ['%gcc@8.3.1', '%clang@10.0.0']
+    - compilers: ['%gcc@8.3.1']
+    
+    # Note skipping spot since no spack package for it
+    - radiuss:
```

Comment:
We'll probably need you to comment some of these out for this PR, so we can grow the stacks a little more slowly.  We're planning to add data+viz sdk first (probably), which is only build `hdf5` at the moment, and see how it goes with the slightly expanded e4s stack coming in #22887.  Then in subsequent PRs, we can uncomment more of the specs in all three stacks.

So, given that, can you find a subset of these specs that results in a DAG with (ballpark) 20 specs total?

Reply:
Sure.  I've been `spack concretize`ing locally so I'll review what I have and see if I can come up with something closer to that number.